src/topology/order/lattice.lean

```diff
@@ -82,3 +85,15 @@ has_continuous_sup.continuous_sup
   {f g : X → L} (hf : continuous f) (hg : continuous g) :
   continuous (λx, f x ⊔ g x) :=
 continuous_sup.comp (hf.prod_mk hg : _)
+
+lemma filter.tendsto.sup_right {ι β} [topological_space β] [has_sup β] [has_continuous_sup β]
+  {l : filter ι} {f g : ι → β} {x y : β}
+  (hf : tendsto f l (𝓝 x)) (hg : tendsto g l (𝓝 y)) :
+  tendsto (f ⊔ g) l (𝓝 (x ⊔ y)) :=
```

Comment:
I think it might be preferred to state these as
```suggestion
  tendsto (λ i, f i ⊔ g i) l (𝓝 (x ⊔ y)) :=
```
Although I'm not sure

Reply:
I think you are right, and I changed the statement to that. I kept the other formulation as a primed lemma.